lib/zip-codes.rb

```diff
@@ -1,5 +1,3 @@
-require 'yaml'
-
 module ZipCodes
   VERSION = '0.2.1'
```

Comment:
This is a breaking change (semver-1)

```suggestion
  VERSION = '1.0'
```

If it's going to be included one day, I think the PR needs to include a build script, that convert a YAML (easy for humans to deal with) into the marshaled format.

Reply:
Feel free to also open a PR against https://github.com/suvie-eng/zip-codes/

I'm not wanting to create maintainer controversy - but until the maintainers here integrate all of these, I'd like to have a gem that is the most up to date.  Also, I like the performance of this change 👌